Modules/Scripted/DICOMLib/DICOMProcesses.py

```diff
@@ -418,8 +418,10 @@ def __init__(self, files, address, protocol=None, progressCallback=None, aeTitle
     def __del__(self):
         super().__del__()
 
-    def defaultProgressCallback(self, s):
+    @staticmethod
```

Comment:
Callback is usually not static method, as it uses other methods of the object to display the progress information, determine if the operation should be cancelled, etc. What was the reason for making it staticmethod?

Reply:
True, this particular function was not using `self`, that was the only reason. But I agree that it should inherit the same architecture -> I'll update this.